prdoc/pr_7719.prdoc

```diff
@@ -0,0 +1,14 @@
+title: Add `export-chain-spec` substrate CLI command
+doc:
+- audience: Runtime Dev
+  description: |-
+    1. Idea is introduce a function export-chain-spec that does the chain-spec exporting. build-spec will work as before, but have a deprecation message on usage.
```

Comment:
Yeah this needsto be more user focused and concise. The PRDoc will be part of the changelog after all.